### ISSUE TITLE
Unable to update id sequences 

### ISSUE THREAD

**Issue by ferrao** [Vincit/knex-db-manager#58]
This package looks awesome and I though it would save me from the dreaded `duplicate key value violates unique constraint` error.

It seems to be properly configured as I can `await dbManager.createDb('some-db');` just fine.
But after inserting some seed data, I try to update the id sequences with `await dbManager.updateIdSequences();`, but nothing seems to happen. 

All the tables keep having have their `id` sequences set to 1, which makes it impossible to insert data after seeding.

**Comment by ferrao** [comment]
I can manually run all those queries and the output seems correct, `knex-db-manager` is getting the exact same list of tables on both `select "tablename" from "pg_tables" where "schemaname" = 'public'` and `select "table_name" from "information_schema"."columns" where "column_name" = 'id'`.

**Comment by ferrao** [comment]
Here goes my code snippet:

```
const dbConfig = {
    client: 'postgresql',
        connection: {
            host: process.env.DB_HOST,
            database: process.env.DB_NAME,
            user: process.env.DB_USER,
            password: process.env.DB_PASS
        },
        pool: {
            min: 2,
            max: 10
        },
        migrations: {
            directory: './db/migrations'
        },
        seeds: {
            directory: './db/seeds/staging'
        },
        acquireConnectionTimeout: 1000,
        debug: {
            query: true,
            tx: true,
            pool: true,
            client: true,
            bindings: true
        }
};

// insert db seeds
const knex = Knex(dbConfig);
await knex.migrate.latest();
await knex.seed.run();
await knex.destroy();

// update id sequences
const dbManager = DbManager.databaseManagerFactory({
    knex: dbConfig,
    dbManager: {
        superUser: 'root'
    }
});
await dbManager.updateIdSequences();
await dbManager.close();
```

**Comment by ferrao** [comment]
Ok, I fixed this issue... 

I was actually using `Objection.knexSnakeCaseMappers` on my knex config, which transformed `table_names` into `tableNames` and cause the error..

Removed the mappers and I am further ahead, but still now working, problem now is:

`error: syntax error at or near "UNION"`

**Comment by ferrao** [comment]
So far I found the following issues:

1) The query fails if a table exists with an `id` column but which has not been configured for auto increment. 

2) The query 
`SELECT 'knex_migrations' AS table, 'public.knex_migrations_id_seq' AS sequence, min_value AS min FROM public.knex_migrations_id_seq;` fails with `ERROR:  column "min_value" does not exist`

The first one is easy to fix @elhigu, but I belive the second is due to the usage of Postgres 10 and the way it handles sequence metadata?

**Comment by ferrao** [comment]
From the Postgres 10 changelog:

> A sequence relation now stores only the fields that can be modified by `nextval()`, that is `last_value`, `log_cnt`, and `is_called`. Other sequence properties, such as the starting value and increment, are kept in a corresponding row of the `pg_sequence` catalog.

**Comment by elhigu** [comment]
Looks like there are actually two postgres tests broken:

    1) #updateIdSequences should update primary key sequences
    2) #updateIdSequences should work with empty table and with minimum value other than 1

I'll give this a maintenance round and make package to work and to be tested on some more modern node versions (and maybe with multiple DB versions) and try to get rid of gatsby which has proven itself to be really hard to maintain.

**Comment by elhigu** [comment]
Fixed in version 0.5.0
